rsconnect/main.py

```diff
@@ -2215,6 +2210,8 @@ def content_describe(
     output_params(ctx, locals().items())
     with cli_feedback("", stderr=True):
         ce = RSConnectExecutor(ctx, name, server, api_key, insecure, cacert, logger=None).validate_server()
+        if not isinstance(ce.remote_server, RSConnectServer):
+            raise RSConnectException("This command requires a Posit Connect server.")
```

Comment:
I used a pretty generic exception message here, but am willing to customize it for each command, if you think that would help.

For example, in this case, it could be:

```
"`rsconnect content describe` requires a Posit Connect server."
```